test/google_lookup_test.rb

```diff
@@ -39,6 +39,33 @@ def test_google_request_denied_read_lat_lon_coordinates
     end
   end
 
+  def test_url_non_enterprise
+    result = lookup.send(:url, '123', '123').split('&')
+
+    assert_equal '/maps/api/timezone/json?location=123%2C123', result[0]
+    assert result[1].start_with?('timestamp=')
+    assert_equal 'key=123abc', result[2]
+    assert_equal 3, result.count
```

Comment:
Please use timecop here to check the timestamp, or just check for the `key` parameter and ignore everything else:

```
result = URI(lookup.send(:url, '123', '123')).query.split('&')
assert result.include?('key=123abc')
```

Ideally `url` won't get tested at all since it's a private method, but I can work on improving the `HTTPTestClient` to deal with that.